### PR TITLE
models: remove Organization and UserOrganization

### PR DESCRIPTION
**Pull request by dinosk**
Connects https://github.com/reanahub/reana-commons/issues/25

Signed-off-by: Dinos Kousidis <dinos.kousidis@cern.ch>

### CODE REVIEW

File: reana_commons/utils.py [hunk 1]
```diff
@@ -52,6 +52,16 @@ def click_table_printer(headers, _filter, data):
         click.echo(formatted_output.format(*row))
 
 
+def get_user_workflows_dir(user):
```
Comment (diegodelemos) [inline reana_commons/utils.py]: This method has been removed in favor of the Users model one: `user_1.get_user_workspace()`. See in [reana-server](https://github.com/reanahub/reana-server/blob/47471d5d58cea0ce78ef73068e10e0199e652349/reana_server/cli.py#L94).

Comment (dinosk) [reply reana_commons/utils.py]: Ah ok, then I'll to change it where it's used as well in r-w-c

**Comment by diegodelemos** [comment]
I think it is just being imported here https://github.com/reanahub/reana-workflow-controller/pull/111/files#diff-b1e90f379b21ae97cefe472ea3b2785bR27.

File: reana_commons/utils.py (hunk 1 above)
Comment (diegodelemos) [reply reana_commons/utils.py]: @dinosk still needs to be removed

**Pull request merged**